docs/querying/multi-value-dimensions.md

```diff
@@ -147,6 +147,12 @@ only row1, and generate a result with three groups: `t1`, `t2`, and `t3`. If you
 your filter, you can use a [filtered dimensionSpec](dimensionspecs.md#filtered-dimensionspecs). This can also
 improve performance.
 
+## Unnesting
+
+You can unnest a column that contains multi-value dimensions (arrays) by using either the [UNNEST function (SQL)](../querying/sql.md#unnest) and the helper function MV_TO_ARRAY or the [`unnest` datasource (native)](../querying/datasource.md#unnest) .
```

Comment:
> ... multi-value dimensions (arrays) ...

please do not conflate multi-value dimensions with actual druid array types, they are totally separate internally. Multi-value strings are represented internally as the native `STRING` type and show up as `VARCHAR` in the SQL schema. Druid _also_ has `ARRAY` types, such as `ARRAY<STRING>`, `ARRAY<LONG>`, etc, that are not well documented yet _on purpose_ to not lock ourselves into a specific behavior prematurely.

Also, its very nearly pointless to use `UNNEST` on a multi-value string column, because all multi-value strings have an implicit unnesting that occurs when grouping. The only case for using on a `STRING` is as part of a scan query